CMakeLists.txt

```diff
@@ -1 +1,2 @@
+zephyr_include_directories(include)
```

Comment:
is this required? (when declaring a dts_root `include` path is automatically added I think)

Reply:
It fails without it.